### PR TITLE
:sparkles: Level 2 and level_transition

### PR DESCRIPTION
**Pull request by GDWR**
The Thing be Done Yes it do be 🥂

### CODE REVIEW

File: game/views/game_view.py [hunk 1]
```diff
@@ -26,9 +26,13 @@ class GameState(Enum):
 
 class GameView(arcade.View):
     door_open_sound = arcade.Sound("game/assets/sound_effects/door_open.wav")
+    level_list = ["level1", "level2"]
```
Comment (SavagePastaMan) [inline game/views/game_view.py:29]: why not just use the int counter we already have and just do something like `f"level{self.level}"`

File: game/views/game_view.py [hunk 2]
```diff
@@ -83,8 +90,9 @@ def load_map(self):
         self.exit_list = arcade.SpriteList()
 
         # Object Layers
-        levelfile = "game/assets/tilemaps/level1_.tmx"
-        self.object_layers = utils.process_objects(levelfile)
+        self.object_layers = utils.process_objects(
+            f"game/assets/levels/{self.level_list[self.level]}.tmx"
```
Comment (SavagePastaMan) [inline game/views/game_view.py:94]: should put this in a variable, you have the exact same fstring above

**Review by SavagePastaMan — CHANGES_REQUESTED** [review]

File: game/views/game_view.py (hunk 1 above)
Comment (GDWR) [reply game/views/game_view.py:29]: sure if you want each level to have that strict naming scheme

File: game/views/game_view.py (hunk 2 above)
Comment (GDWR) [reply game/views/game_view.py:94]: 😨 Where

Comment (GDWR) [reply game/views/game_view.py:94]: oh
yike
sure

**Comment by GDWR** [comment]
too_fast

**Comment by SavagePastaMan** [comment]
Closes #21

**Review by SavagePastaMan — APPROVED** [review]
💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯 💯

**Pull request merged**